docs/3.1. Interoperability - NMOS IS-04.md

```diff
@@ -39,4 +39,4 @@ Where this Device creates a new Source at an Output (see [4.0 Behaviour](4.0.%20
 
 ## Version Increments
 
-In order to prevent unnecessary polling of the Audio Channel Mapping API, any changes to active map parameters belonging to Source connected Outputs MUST be signalled via the IS-04 versioning mechanism by means of a version timestamp increment.
+In order to prevent unnecessary polling of the Audio Channel Mapping API active map resource, any changes to active map parameters belonging to Source connected Outputs MUST be signaled via the IS-04 versioning mechanism by means of a version timestamp increment on the related "source" object.  For active map parameters belonging to outputs which are not backed by an IS-04 registered source, the version timestamp of the related IS-04 device object should be updated.
```

Comment:
I disagree with updating the source id version at all. I believe the sole owner of the mapping resource is the device. I believe the device version number should be updated in all cases when the mapping object has been modified.
Updating the source version number in my opinion would be misleading. As a consumer of the registry subscriptions you would be notified of an update to a resource which has changed no fields and which hosts no external API (so how exactly has the source object changed?)

Reply:
I suppose the question is whether a version increment represents a change to the IS-04 object, or to the fundamental JT-NM Source. My interpretation is the latter - in which case there has been a change to the source as a result of the re-mapping.

Strictly speaking you could probably argue you should actually register a new Source anyway, as changing the content of creates a new Source (it's not the same content any more, so from a JTNM model point of view it's not the same thing any more).